### DISCUSSION TITLE
Show tabs when using Ctrl + Tab in compact mode

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#845] [Ideas]
### What feature would you like to see?

I use zen browser in compact mode and I love the vertical tabs that popout from the left. One issue I have is when I switch tabs there is no visual feedback to know which tab I am on in relation to the order of open tabs. This makes it difficult to navigate to the correct one using Ctrl + Tab and Ctrl + Shift + Tab as I have no idea if the one I want to switch to is in front or behind the current tab. 

This can be solved if the tabs would popout like on mouse hover when switching tabs using keyboard as well.

### What platform are you seeing the problem on?

Linux

### Additional context

Current UX:

[Screencast from 2024-08-21 21-36-21.webm](https://github.com/user-attachments/assets/8830b77f-3ee9-4fed-b687-41bb8ad5e1fb)

Better UX:

[Screencast from 2024-08-21 21-38-08.webm](https://github.com/user-attachments/assets/448e234a-368e-4050-b43c-00b37cced4b9)

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/412_

**Comment by tactipus** [comment]
How would we be able to implement this ourselves?
